### ISSUE TITLE
Mootools enabler/disabler

### ISSUE THREAD

**Issue by gobezu** [gobezu/jbetolo#16]
Excellent example is http://extensions.joomla.org/extensions/core-enhancements/performance/mootools/22236

**Comment by realrandal** [comment]
Yes, it is. But it's maybe better to have one menu list to include/exclude directly in on place in plugin config like Interext:

http://extensions.joomla.org/extensions/core-enhancements/performance/scripts-optimization/18499

It's easier to config from one place and not jump to specific menu positions. But both are great. You want to incorporate one of those solution? Joomla 3.5 will still load mootools from core without need?
